src/ExpressionEvaluator/VisualBasic/Source/ExpressionCompiler/Binders/PlaceholderLocalBinder.vb

```diff
@@ -76,14 +76,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ExpressionEvaluator
             Dim typeChar As String = Nothing
             Dim specialType = GetSpecialTypeForTypeCharacter(identifier.GetTypeCharacter(), typeChar)
             Dim type = Compilation.GetSpecialType(If(specialType = SpecialType.None, SpecialType.System_Object, specialType))
-            ' NOTE: Don't create the local with the canonical name since we want it to have the user's text in diagnostic messages.
+            ' TODO (acasey): don't canonicalize name (GH #878)
```

Comment:
Might be best to expand "GH" for those reading the code long term. Also, no TODOs with aliases.